modules/environment/login/nix-on-droid.nix.default

```diff
@@ -30,23 +30,31 @@
   # Read the changelog before changing this value
   system.stateVersion = "22.05";
 
+  # Set up nix for flakes
+  #nix.extraOptions = ''
+  #  experimental-features = nix-command flakes
+  #'';
+
+  # Set your time zone
+  #time.timeZone = "Europe/Berlin";
+
   # After installing home-manager channel like
   #   nix-channel --add https://github.com/rycee/home-manager/archive/release-22.05.tar.gz home-manager
   #   nix-channel --update
   # you can configure home-manager in here like
-  #home-manager.config =
-  #  { pkgs, lib, ... }:
-  #  {
-  #    # Read the changelog before changing this value
-  #    home.stateVersion = "22.05";
+  #home-manager = {
+  #  backupFileExtension = "hm-bak";
```

Comment:
I'm not sure customizing it is important enough to be prominently presented in the default config.

Reply:
There is no default `backupFileExtension` so activating a home-manager generation could fail if there are any files already present. It is a matter of taste I would say. I can also remove it :)

Reply:
Ah. In that case, I like the default and I think the option is better suited for an advanced template.